### ISSUE TITLE
Support for Laravel 7

### ISSUE THREAD

**Issue by JeffGepiga** [munafio/chatify#7]

**Comment by munafio** [comment]
Can you provide some info about your issue to solve?

**Comment by wikichua** [comment]
@munafio 
1st of all, i have to be honest. this is a great package that i ever observed. I believe it yet to support laravel version 6 ahead due to the pusher package version requirement in ur composer
```json
 "pusher/pusher-php-server": "^3.4@dev"
``` 
would u like to try it out in laravel version 7 with pusher ^4.1?

Thanks

**Comment by wikichua** [comment]
@munafio 
I had tested, this seem workable.
```json
"require": {
        "illuminate/support": "~5|~6|~7",
        "laravel/helpers": "^1.2",
        "pusher/pusher-php-server": "~4.0"
    },
```

**Comment by munafio** [comment]
It will be supported in the next release

**Comment by HDVinnie** [comment]
Why is this closed? It should not be closed until #23 is merged and released.

**Comment by wikichua** [comment]
> Why is this closed? It should not be closed until #23 is merged and released.

@HDVinnie maybe @munafio would want to clean up the backlog for clearer picture to monitor. No worries. I believe his next release will be awesome.

**Comment by munafio** [comment]
@HDVinnie @wikichua This will be fixed in the next release, #23 can not be merged, because it is not the right way to fix this issue.

**Comment by wikichua** [comment]
Noted. Hopefully ur releases will be soon. Excited for ur next release

**Comment by wikichua** [comment]
> @HDVinnie @wikichua This will be fixed in the next release, #23 can not be merged, because it is not the right way to fix this issue.

Ouch.. that's hurt.
